app/controllers/schools/placement_requests/acceptance.rb

```diff
@@ -3,22 +3,15 @@ module PlacementRequests
     module Acceptance
       extend ActiveSupport::Concern
 
-      included do
-        before_action :set_placement_request
-        before_action :fetch_gitis_contact_for_placement_request
-      end
-
       def set_placement_request
```

Comment:
Sorry to be fussy about this but its really important for future developers to be aware an API request is occurring since this is potentially expensive.

If the API request is to be combined into `set_placement_request` (no strong feelings on this), the name does need to reflect that it will query Gitis.

eg `set_placement_request_and_fetch_gitis_contact`